sqlglot/tokens.py

```diff
@@ -186,11 +186,15 @@ def __eq__(self, other):
         if self.token_type == TokenType.STRING:
```

Comment:
might as well not override __eq__ at all if you're going to check line numbers

Reply:
ah, that's true. So we don't really need to normalize any more.